frontend/packages/core/src/AppLayout/colors.tsx

```diff
@@ -0,0 +1,233 @@
+// COLORS FROM FIGMA as of 01-31-2022
+// NOTE THAT THOSE WITH GRADIENTS ONLY HAVE THEIR BASES
+import type { ClutchColorChoices, ClutchPalette } from "./colortypes";
+
+export const LIGHT_MODE: ClutchPalette = {
+  Neutral: {
+    50: "#F8F8F9",
+    100: "#F0F1F3",
+    200: "#DBDBE0",
+    300: "#C2C3CB",
+    400: "#9E9FAC",
+    500: "#868798",
+    600: "#56586E",
+    700: "#3D4059",
+    800: "#252845",
+    900: "#0D1030",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  Blue: {
+    50: "#F9F9FE",
+    100: "#F5F6FD",
+    200: "#EBEDFB",
+    300: "#D7DAF6",
+    400: "#C2C8F2",
+    500: "#727FE1",
+    600: "#3548D4",
+    700: "#1629B9",
+    800: "#0A1CA6",
+    900: "#011082",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  Green: {
+    50: "#E5FCE8",
+    100: "#CBF7CF",
+    200: "#ACF2B2",
+    300: "#6CD47A",
+    400: "#4AB958",
+    500: "#32A140",
+    600: "#1C872A",
+    700: "#106E1D",
+    800: "#086515",
+    900: "#02590E",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  Amber: {
+    50: "#FFFBEB",
+    100: "#FEF3C7",
+    200: "#FDE68A",
+    300: "#FCD34D",
+    400: "#FBBF24",
+    500: "#F59E0B",
+    600: "#D97706",
+    700: "#B45309",
+    800: "#92400E",
+    900: "#78350F",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  Red: {
+    50: "#FFF4F3",
+    100: "#FDDCD2",
+    200: "#F6A996",
+    300: "#F3886E",
+    400: "#F26E50",
+    500: "#F15534",
+    600: "#CA4428",
+    700: "#A1301C",
+    800: "#792111",
+    900: "#571608",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+};
+
+// TODO: come up with a better name
+// The reason why I separated them from above is the above is JUST colors
+export const LIGHT_MODE_CHOICES: ClutchColorChoices = {
+  Stroke: {
```

Comment:
Shouldn't these leverage the color palette so we only need to update one place?

Reply:
oh yeah now I see. For some reason I didn't think of that

Reply:
hm now that I think about this more I think we need to get Kanchan to make some changes. As in, have colors like black / grey / white etc then put those in a palette, then we can reuse them in the color choices object

Reply:
Just found this but looks like MUI supports an alpha functionality to allow for easily changing opacities of theme colors: https://mui.com/guides/migration-v4/#styles

```
import { alpha } from '@mui/material/styles';

 const classes = makeStyles(theme => ({
      backgroundColor: alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
 }));
```